src/app.js

```diff
@@ -12,19 +12,24 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
 */
 "use strict";
 
+var electron = require("electron");
```

Comment:
I think the time has finally come that this single-file app can be more helpfully factored. It looks like there are at least files "menu.js", "tray.js" etc. and see what else you can extract. Also, could you create subdirectories in "src" to separately group together those files which are used in "main" and "renderer" processes. (and a "shared" if there happen to be any definitions shared between the two processes)